src/database/migrations/1742320602314_rename_member_user_public_key_table.ts

```diff
@@ -1,7 +1,6 @@
 import { sql, type Kysely } from 'kysely'
-import { type DB } from '@/database/types'
 
-export async function up(db: Kysely<DB>): Promise<void> {
+export async function up(db: Kysely<unknown>): Promise<void> {
```

Comment:
This shouldn't change any prior migrations but saw [in the docs](https://kysely.dev/docs/migrations#migration-files) they say to explicitly not use your `DB` as the typing, so i reverted these so noone looks at a prior migration file and tries to do the same thing

Reply:
ah interesting, I guess that makes sense